### ISSUE TITLE
Blank dashboard

### ISSUE THREAD

**Issue by code2prog** [laravel/horizon#1029]
- Horizon Version: 5.7.8
- Laravel Version: latest
- PHP Version: 7.4.18
- Redis Driver & Version: predis/phpredis n/a
- Database Driver & Version:

### Description:
Something is wrong. Empty dashboard in version 5.7.8. Updating or publishing assets does not help and only after moving back to 5.7.7 and publishing assets again the dashboard displays correctly

### Steps To Reproduce:
update to 5.7.8

**Comment by edgarsn** [comment]
Same here. Just a blank page. Looks like vue is not initializing or something. No errors in console.

**Comment by driesvints** [comment]
Seems to be working fine for me. Are you sure you published the assets again as described in https://laravel.com/docs/8.x/horizon#upgrading-horizon ?

![Screen Shot 2021-06-03 at 14 04 12](https://user-images.githubusercontent.com/594614/120641973-97fb2080-c474-11eb-976e-f88235f15413.jpg)

**Comment by edgarsn** [comment]
Yeah. `php artisan horizon:publish`. 

Also network tab looks fine as well.

![image](https://user-images.githubusercontent.com/6625918/120643393-ca118000-c47e-11eb-8ad8-7b9fe8db5bf4.png)

Tried on Chrome, Edge & Firefox.

**Comment by driesvints** [comment]
Can you verify that you're using 5.7.8 in your composer.lock file?

**Comment by edgarsn** [comment]
Sure.

<details>
<summary>Part from composer.lock</summary>

```json
  {
      "name": "laravel/horizon",
      "version": "v5.7.8",
      "source": {
          "type": "git",
          "url": "https://github.com/laravel/horizon.git",
          "reference": "2b5628b598c813650458dc7e4afacb500d72ff9a"
      },
      "dist": {
          "type": "zip",
          "url": "https://api.github.com/repos/laravel/horizon/zipball/2b5628b598c813650458dc7e4afacb500d72ff9a",
          "reference": "2b5628b598c813650458dc7e4afacb500d72ff9a",
          "shasum": ""
      },
      "require": {
          "ext-json": "*",
          "ext-pcntl": "*",
          "ext-posix": "*",
          "illuminate/contracts": "^8.17",
          "illuminate/queue": "^8.17",
          "illuminate/support": "^8.17",
          "nesbot/carbon": "^2.17",
          "php": "^7.3|^8.0",
          "ramsey/uuid": "^4.0",
          "symfony/error-handler": "^5.0",
          "symfony/process": "^5.0"
      },
      "require-dev": {
          "mockery/mockery": "^1.0",
          "orchestra/testbench": "^6.0",
          "phpunit/phpunit": "^9.0",
          "predis/predis": "^1.1"
      },
      "suggest": {
          "ext-redis": "Required to use the Redis PHP driver.",
          "predis/predis": "Required when not using the Redis PHP driver (^1.1)."
      },
      "type": "library",
      "extra": {
          "branch-alias": {
              "dev-master": "5.x-dev"
          },
          "laravel": {
              "providers": [
                  "Laravel\\Horizon\\HorizonServiceProvider"
              ],
              "aliases": {
                  "Horizon": "Laravel\\Horizon\\Horizon"
              }
          }
      },
      "autoload": {
          "psr-4": {
              "Laravel\\Horizon\\": "src/"
          }
      },
      "notification-url": "https://packagist.org/downloads/",
      "license": [
          "MIT"
      ],
      "authors": [
          {
              "name": "Taylor Otwell",
              "email": "taylor@laravel.com"
          }
      ],
      "description": "Dashboard and code-driven configuration for Laravel queues.",
      "keywords": [
          "laravel",
          "queue"
      ],
      "support": {
          "issues": "https://github.com/laravel/horizon/issues",
          "source": "https://github.com/laravel/horizon/tree/v5.7.8"
      },
      "time": "2021-05-27T12:07:58+00:00"
  },
```
</details>

**Comment by driesvints** [comment]
So odd. I really wonder what's wrong. Can you maybe try to complete clear your `public/vendor/horizon` directory and try again?

**Comment by edgarsn** [comment]
Same result :/

I can also confirm, downgrading to 5.7.7 fixes problem.

From my side, it looks like something is wrong with javascript intialization. In networks tab there is no `stats`, `workload` or `masters` requests.

**Comment by driesvints** [comment]
I'm afraid that if we can't reproduce this we cannot help. My colleagues also can't reproduce this.

**Comment by edgarsn** [comment]
I will try to debug the source code asap.

**Comment by edgarsn** [comment]
I don't know what I did, but it started to work. First of all I tried to downgrade the package to 5.3.0 which was failing as well for me. That looked very weird at that point. After few composer upgrades/downgrades, I edited `layout.blade.php` file with some random text and everything showed up. 

My guess is - laravel views cache. Recently I was testing production setup (on my development pc) by caching laravel views and probably previous horizon version`s view file was still cached. So it looks like it's not horizon bug, but just simple views cache. Since I'm mostly using laravel as graphql server with only 3 laravel views and frontend is made on nuxt.js, I couldn't see the cache issue at first.

@code2prog Try clearing your laravel cache and check if that helped, especially `php artisan view:clear`.
